src/Driver/Connection.php

```diff
@@ -57,7 +59,7 @@ public function exec(string $sql): int|string;
      * previous statement. DO NOT RELY ON THIS BEHAVIOR which is driver-dependent: always use getLastInsertId() right
      * after executing an INSERT statement.
      *
-     * @return int|string The last insert ID, as an integer or a numeric string.
+     * @return int|numeric-string The last insert ID, as an integer or a numeric string.
```

Comment:
We cannot guarantee that `lastInsertId()` is a numeric string and I doubt that we would gain anything if we could.

Reply:
I just left [a comment](https://github.com/doctrine/dbal/pull/5905#discussion_r1098665008) about this. We should update the description to remove the "numeric" term, right?

Reply:
This is addressed in #5909.